### PR TITLE
[WIP] Configure setup() to use setup.cfg file

### PR DESCRIPTION
**Pull request by pabelanger**
##### SUMMARY
Create setup.cfg file so automation tools can learn more about ansible.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
This affect setup.py 

##### ADDITIONAL INFORMATION
Today zuul will parse the setup.cfg file to learn more abount a python project, this is an example of how we can do more cross-project testing with ansible/ansible.

### CODE REVIEW

**Comment by sivel** [comment]
I don't think we can do this.  As indicated in the commit, this requires setuptools 30.3.0, and Not all distros we support ship with this:

RHEL7: 0.9.8
Ubuntu 16.04: 20.7.0-1

It seems to also be causing some issues in CI as well.

We might be able to have a setup.cfg, but it would have to be in addition to what we already, effectively duplicating information.

**Comment by pabelanger** [comment]
Yah, that's what i wanted to check with CI first. See what breaks, what works.  This is very much WIP ATM

**Comment by sivel** [comment]
@pabelanger Thanks.  I've updated the title to include `[WIP]` so that we have a visual indicator of such.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
```
Comment (danielmellado) [inline setup.cfg:19]: I've seen that tox.ini lacks py37, I assume that as this is a WIP you'll consider adding it there too

**Pull request closed without merging**